piecrust/src/imports.rs

```diff
@@ -217,6 +217,7 @@ fn emit(mut fenv: FunctionEnvMut<Env>, arg_len: u32) {
     env.emit(arg_len)
 }
 
+#[allow(clippy::uninlined_format_args)]
```

Comment:
This does nothing on my machine, what is it for?

Reply:
otherwise Clippy fails in the CI build

Reply:
Please provide the error or link to reproduction.